lib/curation.js

```diff
@@ -17,7 +17,7 @@ class Curation {
   }
 
   static apply(definition, curation) {
-    utils.mergeDefinitions(definition, curation)
+    utils.mergeDefinitions(definition, curation, true)
```

Comment:
instead of overloading utils.mergeDefinitions, can we replace this with lodash merge?

`_.merge(definition, curation)`

Reply:
The problem here is that _.merge is not smart enough to match files by path/hash as per the current structure. This could change if we also applied the change proposed to make file properties key by hash...

Reply:
Ahh makes sense. Thanks for looking into it. We can keep the current implementation then with the overload.